src/integration/sink/testdata/README.md

```diff
@@ -0,0 +1,40 @@
+# Certificates Generation for TLS Tests
+
+## Steps
+
+Install [mkcert](https://formulae.brew.sh/formula/mkcert).
+
+### Generate the Root CA Certificate:
+
+Copy the root CA generated by `mkcert`.
+
+```bash
+cd "`mkcert -CAROOT`"
+cat rootCA.pem
+```
+
+Replace the contents of `rootCA.pem` with the contents of `rootCA.pem`. Do **NOT** copy the key file.
```

Comment:
Can you please rephrase this sentence? It sounds you are replacing contents of a file with its own content, which is a noop. Also, can you explain why not simply copying the key file? There must be a good reason for that

Reply:
I see where the confusion can arise. I'll reword it to use copy instead of replace. Regarding the key file, the engineer needs to copy the CA cert, not the key. The CA key is the private key used to sign certs so committing that would would be bad, hence the warning. I'll clarify this.

Reply:
Thoughts on this: Copy `rootCA.pem` into `src/integration/sink/testdata` and rename it to `rootCA.pem` replacing the expired cert. Do **NOT** copy `rootCA-key.pem` file.

Reply:
So where do you store the key?